### ISSUE TITLE
Transform claim key not found to an empty string.

### ISSUE THREAD

**Issue by guilhermebentomarques** [ThreeMammals/Ocelot#1622]
It's not an issue. It's a question.

Is it possible to check if a key exists and if not pass an empty string?

For example:

What I have: 
`"AddQueriesToRequest": {
        "userId": "Claims[sub] > value"
}`

Error Code: CannotFindClaimError Message: 
Cannot find claim for key: sub errors found in ResponderMiddleware. Setting error response for request

What I would like to do (If the key doesn't exist, send an empty string.
`"AddQueriesToRequest": {
        "userId": "Claims[sub] ? Claims[sub] > value : '' "
}`

Thank you very much.

**Comment by ks1990cn** [comment]
@guilhermebentomarques  No! this breaks functionality. Why you are trying to put claim key after being authenticated? Its not a right idea.

**Comment by ks1990cn** [comment]
@raman-m this can be closed I think, do think it is feasible even?
